Header header
bool state
uint16 signal
